tests/unit/components/molecules/AddTodo.spec.js

```diff
@@ -0,0 +1,94 @@
+import { describe, expect, it } from '@jest/globals'
+import { shallowMount, mount } from '@vue/test-utils'
+import AddTodo from '@/components/molecules/AddTodo.vue'
+import { createStore } from 'vuex'
+import AppCheckboxInput from '@/components/atoms/AppCheckboxInput.vue'
+import AppTextInput from '@/components/atoms/AppTextInput.vue'
+
+describe('Test AddTodo', () => {
+  it('should render', () => {
+    const wrapper = shallowMount(AddTodo, {})
+
+    expect(wrapper.exists()).toBe(true)
+    expect(wrapper.element).toMatchSnapshot()
+  })
+
+  it('should renders InputCheckbox and TextInput components with default props', () => {
```

Comment:
Please split this into two different tests: one for AppCheckboxInput and one for AppTextInput.